src/executables/unloaddb.h

```diff
@@ -59,6 +59,21 @@ extern int lo_count;
 #define PRINT_IDENTIFIER(s) "[", (s), "]"
 #define PRINT_FUNCTION_INDEX_NAME(s) "\"", (s), "\""
 
+/* 
+ * name is user_specified_name.
+ * owner_name must be a char array of size DB_MAX_IDENTIFIER_LENGTH * 2 to copy user_specified_name.
+ * class_name refers to class_name after dot(.).
+ */
+#define SPLIT_USER_SPECIFIED_NAME(name, owner_name, class_name) \
+	do \
+	  { \
+	    memset (owner_name, 0, sizeof (owner_name)); \
+	    memcpy (owner_name, name, strlen (name) + 1); \
```

Comment:
If you must use strlen() anyway, use strcpy()
`strcpy(owner_name, name);`

Reply:
Okay, I'll change it.